### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by DesislavaMetodieva** [heirfanahmed/mood-movies#5]
User Story:
As a web developer, I want to improve the accessibility of the HTML pages so that they can be easily used by a wider range of users, including those with disabilities.

Acceptance Criteria:

All images should have alternative text descriptions.
Proper heading structures should be in place.
Navbar should be wrapped in a header tag.
Navigation should be easily accessible using a keyboard.
Color contrast should meet accessibility standards.
Aria attributes should be properly used for elements that need additional description.
Error messages should be clearly displayed and explained.
The website should be tested with assistive technologies like screen readers. Make sure improve the accessibility of the app with actioning the following points if needed.
Use ARIA: Use the Accessible Rich Internet Applications (ARIA) specification to enhance the accessibility of dynamic content and widgets. Search button should have  aria-label="submit search for forecast location"
